pennylane/tape/measure.py

```diff
@@ -76,6 +78,14 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
         # Queue the measurement process
         self.queue()
 
+    def __copy__(self):
+        cls = self.__class__
+
+        if self.obs is not None:
+            return cls(self.return_type, obs=copy.copy(self.obs))
+
+        return cls(self.return_type, eigvals=self._eigvals, wires=self._wires)
```

Comment:
@co9olguy, I re-wrote the `MeasurementProcess.__copy__` here. I realised this is better logic (it doesn't avoid the `__init__`).